webrtc.html

```diff
@@ -3782,8 +3805,8 @@
                 then perform the following checks:</p>
                 <ul>
                   <li>
-                    <p>If <var>t</var>'s <a data-for=
-                    "RTCRtpTransceiver">direction</a> is "sendrecv" or
+                    <p>If <var>t</var>'s [[<a href=
+                    "#direction-slot">Direction</a>]] slot is "sendrecv" or
```

Comment:
Let's do the "inactive" stuff in a separate issue/PR.

"sendrecv" is a string, but it's also an enum value. We should decide on syntax for enum values and use it everywhere.